appveyor.yml

```diff
@@ -23,14 +23,14 @@ before_build:
   - nuget sources add -name cvodes -source https://ci.appveyor.com/nuget/ospsuite-simmodel-solver-cvodes
   - ps: (get-content src\OSPSuite.SimModelNative\version.h) | foreach-object {$_ -replace "RELEASE 0", "RELEASE $env:RELEASE" -replace "MAJOR 0", "MAJOR $env:MAJOR" -replace "MINOR 0", "MINOR $env:MINOR" -replace "BUILD 0", "BUILD $env:APPVEYOR_BUILD_NUMBER"} | set-content src\OSPSuite.SimModelNative\version.h
   - dotnet restore
+  - dotnet add tests/OSPSuite.SimModel.Tests/OSPSuite.SimModel.Tests.csproj package Appveyor.TestLogger --version 2.0.0
```

Comment:
after some trial and errors: adding this line fixed the issue. Without it I got the error [`dotnet : Could not find a test logger with AssemblyQualifiedName, URI or FriendlyName 'Appveyor'.`](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-simmodel/builds/28184168)
Not sure why this was required. E.g. in the last FuncParser build it was not the case. Maybe AppVeyor VS2019 image has changed since then.